_data/civic_hackers.yml

```diff
@@ -26,6 +26,7 @@ Civic Hackers:
   - open-city
   - openaustralia
   - opendatakosovo
+  - girlscodingkosova
```

Comment:
Nit: think the list is supposed to be alphabetical, but I don't think the CI checks for it right now